### PR TITLE
fix(nodebuilder/p2p): all full and bridge nodes use `dht.ModeServer`

### PR DESCRIPTION
**Pull request by distractedm1nd**
## Overview
Previously, only bootstrappers would use `dht.ModeServer`. Now it is enabled for all full and bridge nodes.

### CODE REVIEW

**Review by Wondertan — APPROVED** [review]
LGTM. @vgonkivs had a concern that the order of those options matters, but I haven't checked it out

**Review by walldiss — APPROVED** [review]

**Comment by distractedm1nd** [comment]
@vgonkivs Should I swap places of the if statements? Will wait for your approval before merging

**Review by renaynay — APPROVED** [review]

**Review by vgonkivs — APPROVED** [review]

**Pull request merged**
